server/api/users.test.js

```diff
@@ -353,3 +353,16 @@ describe("#users.activate", () => {
     expect(body).toMatchSnapshot();
   });
 });
+
+describe("#users.count", () => {
+  it("should return the count", async () => {
+    const user = await buildUser();
+    const res = await server.post("/api/users.count", {
+      body: { token: user.getJwtToken(), id: user.id },
+    });
+    const body = await res.json();
+
+    expect(res.status).toEqual(200);
+    expect(body).toMatchSnapshot();
```

Comment:
Trying to move away from snapshots, they don't make the expectations of the test as explicit

```suggestion
    expect(body.data.count.all). toEqual(1);
    expect(body.data.count.admins). toEqual(0);
    expect(body.data.count.invited). toEqual(0);
    expect(body.data.count.suspended). toEqual(0);
    expect(body.data.count.active). toEqual(1);
```